tests/test_torch_agent.py

```diff
@@ -145,8 +326,8 @@ def test_map_unmap(self):
                         "Incorrectly vectorized text field of obs_batch.")
         self.assertTrue(batch.label_vec is not None, "Missing 'label_vec' field.")
         self.assertTrue(batch.label_vec.numpy().tolist() ==
-                        [[mdict.START_IDX, 7, 8, 9, mdict.END_IDX],
-                         [mdict.START_IDX, 7, 8, 9, mdict.END_IDX]],
+                        [[mdict.BEG_IDX, 7, 8, 9, mdict.END_IDX],
```

Comment:
:'( why tho?

Reply:
I'm having trouble actually pulling up what this was looking at but I removed the code so assuming it's fine :)